spec/controllers/admin/tag_controller_spec.rb

```diff
@@ -247,4 +247,58 @@
       end
     end
   end
+
+  describe 'keep selected filters from index page' do
+    let(:category) { Category.create(name: 'Science') }
```

Comment:
Consider using `model.create!` so that you will see an error when creation of a record fails.